scripts/python/gen.py

```diff
@@ -571,7 +571,8 @@ def launch(self):
             else:
                 self.config_file_path += self.args.config_file_name
 
-            if not os.path.isfile(self.config_file_path):
+            if (not self.args.osinstall and
+                not os.path.isfile(self.config_file_path)):
```

Comment:
Looks like you already fixed this in #413, but this should be indented to avoid the pep8 E122 warnings.